.circleci/config.yml

```
@@ -0,0 +1,89 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+commands:
+  circleci_configure_path:
+    description: Fix for CI prefixing the PATH if you set it via env vars. DO NOT DO THAT. See commit description or details.
+    steps:
+      - run:
+          name: Set the system PATH
+          command: echo "export PATH=/opt/circleci/.pyenv/shims:./bin:./vendor/bin:./submodules/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:./node_modules/.bin" >> ~/.bashrc
+
+  gcloud_configure:
+    description: Configure GCloud
+    steps:
+      - run:
+          name: Configure GCloud
+          command: configure_gcloud.sh
+
+  pull-merge:
+    description: "Bring the repository up-to-date with master and handle setup"
+    steps:
+      - restore_cache:
+          key: v1-yarn-{{ checksum "yarn.lock" }}
+      - run:
+          name: Configure the CircleCI git user
+          command: |
+            git config --global user.email "ci@healthlabs.com"
+            git config --global user.name "Circle CI"
+      - run:
+          name: Configure git to use SSH instead of HTTP
+          command: |
+            git config --global url.git@github.com:.insteadOf git://github.com/
+            git config --global --add url.git@github.com:.insteadOf https://github.com/
+      - run:
+          name: Merge master
+          command: git fetch origin && git merge origin/master --no-edit
+
+  ##################
+  # Docker commands
+  ##################
+  docker_hub_login:
+    steps:
+      - run:
+          name: Login to Docker Hub
+          command: echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+  docker_setup_volumes:
+    steps:
+      - run:
+          name: Setup Docker Volumes
+          command: setup_volumes.sh
+
+
+executors:
+  ex-machina:
+    machine:
+      image: ubuntu-2004:202010-01
+    working_directory: /tmp/stdcheck.com
+
+jobs:
+  fix-and-test:
+    executor: ex-machina
+    steps:
+      - circleci_configure_path
+      - checkout
+      - docker_hub_login
+      - docker_setup_volumes
+      - gcloud_configure
+      - pull-merge
+      - run:
+          name: Intialize Project
+          command: init_submodules_project
```

Is there a reason that we can't use the same convention as the other projects and call this just `init_project`?

```suggestion
          command: init_project
```

---

Since in the other projects, both `bin` and `submodules/bin` is added to PATH, i'm concerned that if we call it init_project it might cause a conflict, since there would be a `bin/init_project `and a `submodules/bin/init_project`, thats it two init_project in PATH